src/main/java/de/craften/ui/swingmaterial/MaterialMenuItem

```diff
@@ -0,0 +1,59 @@
+package de.craften.ui.swingmaterial;
+
+import de.craften.ui.swingmaterial.MaterialShadow;
+import de.craften.ui.swingmaterial.fonts.Roboto;
+import java.awt.Color;
+import java.awt.Dimension;
+import javax.swing.Action;
+import javax.swing.BorderFactory;
+import javax.swing.Icon;
+import javax.swing.JMenuItem;
+import javax.swing.UIManager;
+import javax.swing.plaf.basic.BasicMenuItemUI;
+
+/**
+ *
+ * @author Rodrigo Butzke
+ */
+public class MaterialMenuItem extends JMenuItem {
+
+    public MaterialMenuItem() {
+        init();
+    }
+    public MaterialMenuItem(Icon icon) {
+        super(icon);
+        init();
+    }
+    public MaterialMenuItem(String text) {
+        super(text);
+        init();
+    }
+    public MaterialMenuItem(Action a) {
+        super(a);
+        init();
+    }
+    public MaterialMenuItem(String text, Icon icon) {
+        super(text, icon);
+        init();
+    }
+    public MaterialMenuItem(String text, int mnemonic) {
+        super(text, mnemonic);
+        init();
+    }
+    private void init() {
```

Comment:
As already stated in the other PR, please don't to this. Move the initialization logic into a constructor and call that constructor from all other constructors.